nano/node/socket.hpp

```diff
@@ -13,6 +13,17 @@
 
 namespace nano
 {
+/** Policy to affects at which stage a buffer can be dropped */
+enum class buffer_drop_policy
+{
+	/** Can be dropped by bandwidth limiter (default) */
+	limiter,
+	/** Should not be dropped by bandwidth limiter */
+	no_limiter_drop,
+	/** Should not be dropped by socket write queue limiter */
```

Comment:
Also not dropped by the bandwidth limiter right?

Reply:
Correct, will clarify in comment as well, thanks.